wrangler-service/src/main/java/co/cask/wrangler/service/FileTypeDetector.java

```diff
@@ -46,7 +46,7 @@ public FileTypeDetector() {
       Scanner scanner = new Scanner(file);
       while(scanner.hasNext()) {
         String line = scanner.nextLine();
-        String[] parts = line.split("[\t+\\s+]");
+        String[] parts = line.split("\\t+");
```

Comment:
Why the change?

Reply:
This is from a comment I made in the JIRA that the file.extensions mapping file uses tabs almost everywhere but spaces in a couple places. Wasn't sure if the ones with spaces were getting ignored or not. 

Maybe just use `\\s+` to be more safe? The previous regex is silly, a tab counts as whitespace.

Reply:
Split by whitespaces `\\s+` seems make more sense.